src/core/api/public_api.ts

```diff
@@ -1288,6 +1313,19 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     return this.videoElement.currentTime;
   }
 
+  /**
+   * Get the last stored content position, in seconds.
+   *
+   * @returns {Number}
+   */
+  getLastStoredContentPosition() : number {
+    const reloadPosition = this._priv_reloadingMetadata.reloadPosition;
+    if (reloadPosition === undefined) {
+      throw new Error("No previously loaded content.");
```

Comment:
Yes I could think of multiple cases where `0` would not be the best:
  - For live contents, we do not begin at `0`, but the application may decide to seek at `0` at one point. So here the application wouldn't be able to tell easily which situation it is in
  - Even for VoD contents, an application may decide to start from some point on (for example because the user re-watch from where he/she previously stopped)

From here, we could modify this function by returning by default the initial position instead in general, and why not, but there would still be a difficult-to-define case for when no content at all were loaded before.

Though it's true that we could be returning `0` only in that last case, but I thought that returning `undefined` for when it doesn't make sense seems more explicit and differentiating, giving more power to the application if for example it wants to only do things if a previous content was loaded, or for example to either reload a content from its last loaded position or from a different, non-0, initial position.

Reply:
Going for `undefined` then!